libs/wingsdk/src/cloud/function.ts

```diff
@@ -30,6 +30,12 @@ export interface FunctionProps {
    * @default 1m
    */
   readonly timeout?: Duration;
+
+  /**
+   * The amount of memory to allocate to the function, in MB.
+   * @default 128
```

Comment:
Is there a max value as well? Do we want to force that max?

Reply:
Great idea -- for any cloud targets that support his field, we should try and check that the user passes a valid input. We should start by checking the AWS docs and see what are the constraints are (like minimum and maximum) and see if they can be implemented as code